exercises/phone-number/examples/success-standard/src/Phone.hs

```diff
@@ -1,9 +1,18 @@
 module Phone (number) where
-import Data.Char (isDigit, isLetter)
+import Data.Char (isDigit)
 
 number :: String -> Maybe String
-number input
-  | any isLetter input = Nothing
+number input = clean input >>= check
```

Comment:
You code is definitely clearer!

If I where to rewrite it, I would probably *"point-free-fy it"* with `number = clean >=> check`, but it has the inconvenience of an additional import.

Does it type-checks?